src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/chatpageobject/ChatPageObject.java

```diff
@@ -59,6 +61,24 @@
   private WebElement chatLoadedIndicator;
   @FindBy(css = "#ChatHeader h1.private")
   private WebElement privateChatHeader;
+  @FindBy(css = "div.actions .admin-actions .give-chat-mod")
+  private WebElement addUserModStatusButton;
+  @FindBy(css = "div#UserStatsMenu.UserStatsMenu")
+  private WebElement userOptions;
+  @FindBy(css = "ul.admin-actions")
+  private WebElement userModOptions;
+  @FindBy(css = "li.message-wall")
+  private WebElement userMessageWallButton;
+  @FindBy(css = "textarea#WallMessageTitle.title")
+  private WebElement userPageMessageWallTab;
+  @FindBy(css = "li.contribs")
+  private WebElement userContributionsButton;
+  @FindBy(css = "table.mw-contributions-table")
+  private WebElement userPageContributionsTab;
+  @FindBy(css = "form#Write.Write.blocked")
+  private WebElement userBlockedMessageField;
+  @FindBy(css = "div.header-column.header-title")
+  private WebElement PermissionsErrorTitle;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this field "PermissionsErrorTitle" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00116)